versions/1.1.0-dev.md

```diff
@@ -141,6 +142,25 @@ The properties of the resolved `copy` object MUST be compatible with the target
 
 This object MAY be extended with [Specification Extensions](#specification-extensions).
 
+### Parameter Object
+
+The object represents parameters that can be used with actions.
+
+#### Fixed Fields
+
+| Field Name | Type | Description |
+| ---- | :----: | ---- |
+| <a name="parameter-name"></a>name | `string` | **REQUIRED** A name for the parameter that is unique for the action the parameter is defined. The name MUST match the following regular expression `[a-zA-Z-0-9]+`. |
+| <a name="parameter-source"></a>source | `inline` or `environment` | **REQUIRED** Defines where the values for the parameters should be retrieved from. When `inline` is set, values are retrieved from the `values` property. When `environment` is set, the values are retrieved by the tool from environment variables. |
+| <a name="parameter-values"></a>values | `string[]` | **REQUIRED** when `source` is set to `inline`. MAY contain zero or more entries. When values contains multiple entries, the action is effectively projected for each of the entries of the parameter. If multiple parameters are defined for any given action, parameters effectively act as a matrix definition and the action should be projected across each dimension. |
+| <a name="parameter-separator"></a>separator | `string` | Only applicable when `source` is set to `environment`. Defaults `undefined` (no splitting is applied). When defined, the separator is used to split the environment variable value and extra multiple values. |
+
+#### Parameter Expressions
+
+Parameter expressions MAY be used in any field defined by the Action Object except for parameters. The syntax for parameter expressions is the following `${parameter_name}`. ONLY parameter expressions with a matching parameter definition for the action MAY be replaced by their values.
```

Comment:
Why the dollar before the curlies? OpenAPI path template parameters and server variables only use curlies around the parameter name.

Reply:
I'm not married to this specific syntax. I want to maintain a syntax that's simple enough it can be implemented via string replace, not even regex if we can.

Also that syntax needs to be different ythe path template sin OAI so implementers of overlay parameters can tell the difference in that specific scenario.